src/main/resources/localization/messages-pl.properties

```diff
@@ -0,0 +1,78 @@
+error.permissions.user = Aby użyć tej komendy potrzbujesz permisji Manage Server. 
+error.permissions.perms_or_role = Aby użyć tej komendy potrzebujesz permisji Manage Server albo roli {0}. 
+error.permissions.perms_no_role = Aby użyć tej komendy potrzbujesz permisji Manage Server. Rola zarządcy giveawayi nie została jeszcze ustawiona. Administrator może ją ustawić za pomocą `{0}`
+error.permissions.bot = Na tym kanale brakuje mi tych permisji: `Read Messages`, `Send Messages`, `Read Message History`, `Embed Links`. Napraw te permisje albo zaproś mnie jeszcze raz [tutaj]({0}).
+error.invalid_time.format = Nie mogłem zamienić `{0}` na poprawną ilość czasu! 
+error.invalid_time.min = Podany czas ({0}) jest krótszy niż czas minimalny ({1})!
+error.invalid_time.max = Podant czas ({0}) jest dłuższy niż czas maksymalny ({1})!
+error.max_giveaways.guild = Istnieją już {0} aktywne giveawaye an tym serwerze!  
+error.max_giveaways.channel = Istnieją już {0} aktywne giveawaye an tym kanale!
+error.invalid_winners.format = Nie mogłem zamienić `{0}` na poprawną ilość wygranych!
+error.invalid_winners.max = Podana ilość wygranych ({0}) nie mieście się między 1, a {1}!
+error.invalid_prize.length = Długość nagrody nie może być dłuższa niż {0} znaków! 
+error.invalid_descr.length = Długość opisu nie może być dłuższa niż {0} znaków!
+error.invalid_emoji.length = Tekst customowego przycisku nie może być dłuższa niż {0} znaków!
+error.invalid_emoji.choice = W pełni customowe przyciski giveawayi są dostępne tylko w planie premium. Użytkownicy bez planu premium mogą użyć jakiejkolwiek z następujących emotek: {0}
+error.invalid_id = Nie mogłem zamienić `{0}` na poprawne ID wiadomości! 
+error.message_not_found = Nie mogłem znaleźć wiadomości o ID `{0}` na tym kanale!
+error.invalid_message = Ta wiadomość nie jest wiadomością zakończonego gievawaya!
+error.giveaway_not_found = Nie mogłem znaleźć giveawaya o ID `{0}`!
+error.giveaway_already_entered = Już dołączyłeś do tego giveawaya!
+error.giveaway_not_entered = Jeszcze nie dołączyłeś do tego giveawaya!
+error.no_dms = Ten bot nie może być używany w prywatnych wiadomościach!
+error.giveaway_cooldown = Tworzenie giveawaya jest obecnie niedostępne ze względu na cooldown!
+error.giveaway_ended = Nie możesz dołączyć ani opuścić tego giveawaya, gdyż ten już się zakończył!
+error.invalid_color = Nie mogłem zamienić `{0}` na właściwy kolor! 
+error.reroll_message_option = Zwycięzcy giveawaya mogą być także wylosowani ponownie poprzez kliknięcie prawym (komputer) / kliknięcie i ptrzytrzymanie (telefon) na oryginalnej wiadomości giveawaya, następnie naciśnięcie na **Apps** > **Reroll Giveaway**
+error.generic.ending = Wystapił problem podczas zakańczania giveawaya.
```

Comment:
Correct the word "zakańczenia" to "zakończenia"

Reply:
"Zakończenia" would imply that it was something already done, since the error occured while ending the giveaway the action wasn't complete and thus "zakańczania" is the correct form here.